### ISSUE TITLE
questions about traing time

### ISSUE THREAD

**Issue by tb2-sy** [ActiveVisionLab/nope-nerf#5]
Nice work! The paper does not seem to mention the training time and the nvidia graphics cards you use, can you tell me about this information, thanks.

**Comment by bianwenjing** [comment]
Hi, thanks for your interest in our work.
The training time varies for different scenes. Here is the time on the scene ‘Ignatius’ (105 training images) for your reference. Total training time: ~38 hours, 1.36M iterations. As we manually set the scheduling iterations, it can be longer than the actual time needed for the poses and NeRF to converge. We use a single TITAN RTX for training.

**Comment by emiald** [comment]
I training it almost 10000epoch,the PSNR impored so slow.................................

**Comment by bianwenjing** [comment]
Hi, as we do not decay the learning rate from the beginning, the PSNR is not likely to increase much before the lr scheduler start, which leads to longer training time. The purpose of doing this is to avoid overfitting the NeRF model before obtaining accurate poses. In practice, it may be unnecessary for some scenes. You can customise when the learning rate scheduler starts by setting`cfg['training']['auto_scheduler']` to `False` and manually defining `cfg['training']['scheduling_start']` (in epochs), for example, to 100.

**Comment by engrmusawarali71** [comment]
@bianwenjing  what does it mean by the images saved in rendering folder?
